internal/api/api.go

```diff
@@ -33,6 +34,9 @@ import (
 	intupload "github.com/artefactual-sdps/enduro/internal/upload"
 )
 
+// go:embed gen/http/*.*
```

Comment:
I think that the main issue is in that extra space after `//`. It should read:

```go
//go:embed gen/http/*.*
```

But you probably want something like this to embed just that file:

```go
//go:embed gen/http/openapi.json
```

Reply:
Done. :)